src/DIRAC/Core/DISET/private/MessageBroker.py

```diff
@@ -7,11 +7,16 @@
 __RCSID__ = "$Id$"
 
 import threading
-import select
 import time
+import select
```

Comment:
It does not seem needed anymore.

Reply:
There is still the `except select.error`. I suspect it shouldn't be there but I'm not confident enough to remove it.